tapalcatl_server/log.go

```diff
@@ -111,6 +119,22 @@ func (l *JsonLoggerImpl) Metrics(metricsData map[string]interface{}) {
 	l.Log(metricsData)
 }
 
+func (l *JsonLoggerImpl) ExpVars() {
+	var buffer bytes.Buffer
+	buffer.WriteString("{")
+	first := true
+	expvar.Do(func(kv expvar.KeyValue) {
+		if first {
+			first = false
+		} else {
+			buffer.WriteString(",")
+		}
+		fmt.Fprintf(&buffer, "\"%s\":%s", kv.Key, kv.Value.String())
+	})
+	buffer.WriteString("}")
+	l.Logger.Printf("{\"type\":\"info\",\"category\":\"%s\",\"expvars\":%s}\n", LogCategory_ExpVars.String(), buffer.String())
```

Comment:
The `kv.Value.String()` is already json encoded, so if we stick that in a map we get it double escaped. This is the approach taken in the [golang source](https://golang.org/src/expvar/expvar.go) too.